### PR TITLE
Beta Version - WIP

### PR DESCRIPTION
**Pull request by db80**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        value: github.com/ennismore/*
+      GITHUB_TOKEN:
+        from_secret: GITHUB_EM_GCR_TOKEN
+      ENDPOINT: http://elasticmq:19324
+
+    # 2. publish to ECR
+  - name: publish-ecr
+    image: plugins/ecr
+    settings:
+      repo: 673749904140.dkr.ecr.eu-west-2.amazonaws.com/em-booking-delete
```
Comment (mryan321) [inline .drone.yml:42]: will need creating: https://eu-west-2.console.aws.amazon.com/ecr/repositories?region=eu-west-2

File: .terraform-config/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,9 @@
+app_name = "bookingdelete"
```
Comment (mryan321) [inline .terraform-config/terraform.tfvars:1]: this should be gitignored - also dont need the db creds, no db 🤔

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM golang:1.13-alpine AS builder
```
Comment (mryan321) [inline Dockerfile:1]: we could use the slightly thinner version: https://github.com/ennismore/em-config/blob/develop/Dockerfile - no external https comms 🤔

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	github.com/ennismore/awssdk-go v0.0.5
+	github.com/ennismore/em-domain/v2 v2.0.61
+	github.com/golang/protobuf v1.4.0
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/sirupsen/logrus v1.6.0
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
+	google.golang.org/grpc v1.27.1
+)
+
+replace github.com/ennismore/em-domain/v2 => /var/www/ennismore/em-domain
```
Comment (mryan321) [inline go.mod:16]: delete me

File: go.mod [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+replace github.com/ennismore/awssdk-go => /var/www/ennismore/awssdk-go
```
Comment (mryan321) [inline go.mod:18]: delete me

File: main.go [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"os"
+	"os/signal"
+	"strconv"
+	"time"
+)
+
+type Config struct {
+	EmApiPort   int    `envconfig:"EM_API_PORT"`
```
Comment (mryan321) [inline main.go:18]: can use split words - also think we should define localhost defaults as standard 🤔  e.g. `PostgresUrl      string `default:"postgres://hasura:hasura@localhost:5432/hasura?sslmode=disable" split_words:"true"`` (https://github.com/ennismore/em-config/blob/6f950e08659fccde3f252249f68ab41a5fdb241f/main.go#L22)

**Review by mryan321 — COMMENTED** [review]
just some comments from quick eyeball :)

File: Dockerfile (hunk 3 above)
Comment (mryan321) [reply Dockerfile:1]: ignore this - we comm with AWS 🙈  😄

**Review by mryan321 — APPROVED** [review]
older comments still apply but lgtm for main part 👍😄

File: .kube-config/config-dev.yml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: em-booking-delete
+  namespace: default
+data:
+  log-level: "debug"
+  em-api-port: "https://sqs.eu-west-2.amazonaws.com"
```
Comment (mryan321) [inline .kube-config/config-dev.yml:8]: do we need these em-api ones? they look wrong vals too - but will be same for all envs so  hardcode in deployment.yaml as a value 🤔

**Review by mryan321 — APPROVED** [review]

File: .kube-config/config-dev.yml (hunk 7 above)
Comment (db80) [reply .kube-config/config-dev.yml:8]: :/ my bad

**Pull request closed without merging**